patroni/__init__.py

```diff
@@ -164,7 +167,28 @@ def shutdown(self):
 
 
 def patroni_main():
-    patroni = Patroni()
+    config = __import__('patroni.config', fromlist=[""])
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+            '--version', action='version',
+            version='%(prog)s {version}'.format(version=__version__)
+            )
+    parser.add_argument(
+            "configfile",
+            nargs="?",
+            default="",
+            help="Patroni may also read the configuration from the {0} environment variable"
+            .format(config.Config.PATRONI_CONFIG_VARIABLE)
+            )
+    args = parser.parse_args()
+    try:
+        conf = config.Config(args.configfile)
+    except ConfigParseError as e:
```

Comment:
If the config file is broken/malformed there might be some other exceptions, for example:
```python
    self.peek_token().start_mark)
yaml.parser.ParserError: expected '<document start>', but found '<block mapping start>'
  in "postgres1.yml", line 3, column 1
```

Reply:
Do we want to do it in this PR? we didn't bother to catch those errors before. We can catch errors but it seems out of scope of this task.